src/main/java/us/abstracta/wiresham/ConnectionFlowStepsDriver.java

```diff
@@ -4,26 +4,28 @@
 import java.net.Socket;
 import java.nio.ByteBuffer;
 import java.util.LinkedList;
+import java.util.List;
 import java.util.Queue;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.slf4j.MDC;
 
 /**
- * Drives the flow of a connection according to a configured flow.
+ * Drives the flow of a connection according to configured steps.
```

Comment:
Any reason for the rename?, is there any change in concepts? Isn't a flow a collection of steps?